packages/gatsby-remark-prismjs/src/highlight-code.js

```diff
@@ -34,7 +36,7 @@ module.exports = (
         console.warn(message, `applying generic code block`)
         unsupportedLanguages.add(lang)
       }
-      return _.escape(code)
+      return escapeHTML(_.escape(code), additionalEscapeCharacters)
```

Comment:
This means we escape stuff twice, I feel it would be better if our escapeHTML function also handled what lodash escape currently handles.

Reply:
totally agree. I felt weird about this and just wanted another set of eyes to confirm.

Reply:
I've modified the routine so that it uses lodash's escape behaviour. This seemed like not a very heavy lift. From what I can tell, the library is not being used anymore so i'd suggest stripping it out?